pkg/trace/config/config.go

```diff
@@ -594,7 +592,7 @@ func (c *AgentConfig) AllFeatures() []string {
 
 //nolint:revive // TODO(APM) Fix revive linter
 func InAzureAppServices() bool {
```

Comment:
Can this now be unexported? (ie lowercased)

Reply:
Yes! And I can remove it from the struct too. Thanks!